config/environments/development.rb

```diff
@@ -15,7 +15,7 @@
 
   # Don't care if the mailer can't send
   config.action_mailer.raise_delivery_errors = true
-  config.action_mailer.delivery_method = :test
+  config.action_mailer.delivery_method = :letter_opener
   config.action_mailer.perform_deliveries = false
```

Comment:
I cracked the gem open.  It appears we will have to call `.deliver` for the gem to work.  And I also had to set `perform_deliveries = true` in `config/environments/development.rb`.